low-level/sspl_ll_d

```diff
@@ -715,6 +717,15 @@ def send_thread_controller_request(state):
 #     logger.info(f"Received SIGUSR1 signal to set log level to '{log_level}'")
 #     logger.setLevel(log_level)
 
+def initialize_iem():
+    try:
+        EventMessage.init(component='sspl', source='S')
+        logger.debug("IEM framework initiated successfully!!!")
+    except EventMessageError as e:
+        logger.error("Failed to initialize IEM Framework."
+                     f"ERROR:{e}")
+        raise
```

Comment:
This would make sspl exit with traceback, since no code is handling exception from this function ?

Reply:
done the change